File: Python/practical/6c.py
# Program to check whether number is even or odd

def typeChk(num):
    if num % 2 == 0:
        return True
    else:
        return False

num = int(input(f"Enter the number: "))

if typeChk(num) == True:
    print(f"{num} is an even number")
else:
    print(f"{num} is an odd number")

File: C/SEM I/Practical/Done/6c.c
/* Write a C program to check whether a number is even or odd */

#include <stdio.h>
#include <conio.h>
#include <stdlib.h>

int count = 0;

int main(void) {
	int rep = 1;

	while (rep == 1) {
		int num;

		printf("Enter the number to check: ");
		scanf("%d", & num);

		if (num % 2 == 0) {
			printf("%d is even\n", num);
		} else if (num % 2 != 0) {
			printf("%d is odd\n", num);
		}

		printf("\nRepeat Program?: ");
		scanf("%d", & rep);

		count++;

		if (count >= 50) {
			printf("Abnormal END\n");

			getch();
			exit(0);
		}

	}

	printf("Normal END\n");

	getch();
	return 0;
}